### PR TITLE
enable eeprom

### PR DESCRIPTION
**Pull request by JyeSmith**
Thanks PeHo :muscle:

Slightly changed writeEEPROM() condition to only write after the last change.  Previously it would write immediately after the first of multiple changes and then again 1s later after more changes.

Pitmode logic is a little broken on boot and needs addressing.

### CODE REVIEW

File: src/src/common.c [hunk 1]
```diff
@@ -8,9 +8,8 @@
 
 uint8_t rxPacket[16];
 uint8_t txPacket[18];
-uint8_t vtxModeLocked;
+uint8_t vtxModeLocked = 0;
```
Comment (cruwaller) [inline src/src/common.c:11]: No need to init zero. That is done automatically at start up phase (check bss section :) ).
No need to fix now!

File: src/src/openVTxEEPROM.h [hunk 2]
```diff
@@ -2,11 +2,11 @@
 
 #include <stdint.h>
 
-#define versionEEPROM 0x110
+#define versionEEPROM 0x0001
 
 #define BOOT_FREQ 5000
 
-extern uint8_t updateEEPROM;
+extern uint32_t eeprom_last_write;
```
Comment (cruwaller) [inline src/src/openVTxEEPROM.h:9]: This should not be available outside. Remove this.

File: src/src/smartAudio.c [hunk 3]
```diff
@@ -145,8 +145,6 @@ void smartaudioBuildSettingsPacket(void)
 
 void smartaudioProcessFrequencyPacket(void)
 {
-    initFreqPacketRecived = 1;
```
Comment (cruwaller) [inline src/src/smartAudio.c:148]: Most likely not related to eeprom. Why this is removed?

Comment (cruwaller) [reply src/src/smartAudio.c:148]: Oh, I see now. Ignore this!

File: src/src/common.c (hunk 1 above)
Comment (JyeSmith) [reply src/src/common.c:11]: I knew you wouldn't like this LOL

At the time I did it for my own readability.  Ill revert it.

File: src/src/openVTxEEPROM.h [hunk 4]
```diff
@@ -32,6 +30,7 @@ typedef struct
 } openVTxEEPROM;
 
 extern openVTxEEPROM myEEPROM;
+uint32_t eeprom_last_write;
```
Comment (cruwaller) [inline src/src/openVTxEEPROM.h:33]: No definition in header. This causes build failure :)
Remove this line

**Comment by JyeSmith** [comment]
Whoops, I accidentally included a TRAMP fix in the last commit.  When exiting pitmode it was using setPowerdB with mW and continually setting the VTx to max output!

**Review by cruwaller — APPROVED** [review]

**Pull request merged**